### PR TITLE
Fixing PHPCS filset reference.

### PR DESCRIPTION
**Pull request by dpagini**
I'm not sure if this is correct, but it seems like build.yml has a mistake referencing a fileset for code sniffing. The `files.frontend.custom.themes` fileset contains javascript, bower, npm, etc. files, not PHP code. Seems odd to send that through code sniffing. Should this be the `files.php.custom.themes` fileset instead?

### CODE REVIEW

**Comment by grasmash** [comment]
JS and CSS code can be sniffed via PHPCS using Drupal coding standards.

**Comment by dpagini** [comment]
Ah ha, interesting. Should this have both lines then? Shouldn't `files.php.custom.theme` be added as well?

**Comment by grasmash** [comment]
Yes I think that's right!

**Comment by dpagini** [comment]
Updated. Thanks @grasmash.

File: phing/build.yml [hunk 1]
```diff
@@ -91,7 +91,8 @@ phpcs:
   filesets:
     - files.php.custom.modules
     - files.php.tests
-    - files.frontend.custom.themes
+    - files.php.custom.themes
+    - files.frontend.custom.theme
```
Comment (grasmash) [inline phing/build.yml]: Looks like you accidentally removed the s.

Comment (dpagini) [inline phing/build.yml]: Doh! Fixing now.

**Pull request merged**